scripts/python/main.py

```diff
@@ -0,0 +1,68 @@
+'''
+
+ 
+ File: 
+   main.py
+
+
+ Description:
+   A driver for the TLE orbit plotter.
+
+
+ Record of Revisions:
+
+   Date:         Author:          Description:
+   2022-02-06    P. Stegmann      Original Code
+
+
+'''
+
+import numpy as np
+import urllib.request              # Downloading NORAD data
+import orbit_gnss as gnss
+import orbit_graphics
+
+if __name__ == "__main__":
+    # elem1 = """ISS (ZARYA)
+    # 1 25544U 98067A   08264.51782528 -.00002182  00000-0 -11606-4 0  2927
+    # 2 25544  51.6416 247.4627 0006703 130.5360 325.0288 15.72125391563537"""
+
+    # dragon = """DRAGON CRS-2            
+    # 1 39115U 13010A   13062.62492353  .00008823  00000-0  14845-3 0   188
+    # 2 39115  51.6441 272.5899 0012056 334.2535  68.5574 15.52501943   306"""
+
+    orbit_graphics.plotEarth()
+
+    gps_orbit = gnss.orbit("gps-ops")
+
+    # Data from NORAD http://www.celestrak.com/NORAD/elements/
+    # filename = "noaa.txt" # NOAA satellites
+    # filename = "geo.txt" # Geostationary satellites
+    # filename = "gps-ops.txt" # GPS sats
+    # filename = "military.txt" # Some military satellites
+    # filename = "stations.txt" # Space stations
+    # filename = "visual.txt" # 100 brightest or so objects
+
+    # files = ["noaa.txt", "stations.txt", "military.txt", "gps-ops.txt"]
+    # files = ["stations.txt"]
+
+    names = ["gps-ops"]
+
+    for urlname in names:
+        f = urllib.request.urlopen("http://www.celestrak.com/NORAD/elements/%s.txt" % urlname)
+        elem = ""
+        for line in f:
+            line = str(line.decode('UTF-8'))
+            print(line)
+            elem += line
+            if (line[0] == '2'):
+                elem = elem.strip()
+                print(elem)
```

Comment:
was printing the following:

GPS BIII-3  (PRN 23)
1 45854U 20041A   22038.44237535 -.00000055  00000+0  00000+0 0  9990
2 45854  55.4238  91.4915 0019468 175.7803 312.9411  2.00560548 12131
GPS BIII-4  (PRN 14)

1 46826U 20078A   22038.03999833 -.00000014  00000+0  00000+0 0  9996

2 46826  54.6537 275.3858 0013962 182.5691 347.5095  2.00566642  9596

GPS BIII-4  (PRN 14)
1 46826U 20078A   22038.03999833 -.00000014  00000+0  00000+0 0  9996
2 46826  54.6537 275.3858 0013962 182.5691 347.5095  2.00566642  9596

Reply:
Right. I have cleaned up the print statements to reduce the output clutter.

Reply:
Nice and clean!

semi_major_axis = 26560.4km
eccentricity    = 0.0056176
inclination     = 55.4949°
arg_perigee     = 53.8316°
right_ascension = 160.773°
true_anomaly    = -279.751°
----------------------------------------------------------------------------------------
GPS BIIR-2  (PRN 13) : Lat: -48.3961° Long: -50.7274
GPS BIIR-2  (PRN 13)
1 24876U 97035A   22037.73434929  .00000020  00000+0  00000+0 0  9996
2 24876  55.4949 160.7731 0056176  53.8316 306.6658  2.00563345180035